src/app/game-state/home.service.ts

```diff
@@ -720,9 +724,6 @@ export class HomeService {
         // autoBuy is on, we have enough land, check if we have the money for the house plus food and rent
         if ((this.characterService.characterState.money > this.nextHome.cost + priceBuffer )){
           this.upgradeToNextHome();
-        } else {
```

Comment:
It _seems_ like this isn't necessary, since the other things check for it first, and because house is first in the list here. Returning here would prevent buying, say, a 10 tael blanket when we don't have enough money to purchase a mansion.

Reply:
Yeah, I think I wrote this part before furniture existed and never updated it. Kind of feeds into the feature request to allow the user to specify the autobuy priorities. I'm fine with removing the return for now.